.github/ISSUE_TEMPLATE.md

```diff
@@ -1,3 +1,3 @@
-THIS ISSUE TRACKER IS CLOSED - please log new issues here: https://github.com/aspnet/Home/issues
+THIS ISSUE TRACKER IS CLOSED - please log new issues here: https://github.com/aspnet/AspNetCore/issues
```

Comment:
Isn't this repo becoming Extensions and tracking its own bugs?

Reply:
I believe the plan was to still use aspnet/AspNetCore for now, but I personally think it would make sense to re-open this repo's issue tracker.

Reply:
Oh yeah I thought we were going to have an issue tracker in each new mondo repo.

Reply:
Ok. I'll remove this and copy in issue templates from aspnet/AspNetCore instead.